static-src/scripts/App.js

```diff
@@ -30,7 +27,5 @@ module.exports = App.extend({
       hashChange: true
     });
     */
-
-  }
-
+  },
```

Comment:
Oh, and AirBnB style guide prefers trailing commas everywhere (for prettier git diffs). I'm not a huge fan, but what can you do...